packages/calypso-products/src/translations.tsx

```diff
@@ -495,6 +495,276 @@ export const getJetpackProductsFeaturedText = (): Record< string, TranslateResul
+		[ PRODUCT_JETPACK_VIDEOPRESS_MONTHLY ]: videoPressIncludesInfo,
+		[ PRODUCT_JETPACK_ANTI_SPAM ]: antiSpamIncludesInfo,
+		[ PRODUCT_JETPACK_ANTI_SPAM_MONTHLY ]: antiSpamIncludesInfo,
+		[ PRODUCT_JETPACK_SCAN ]: scanIncludesInfo,
+		[ PRODUCT_JETPACK_SCAN_MONTHLY ]: scanIncludesInfo,
+		[ PRODUCT_JETPACK_SCAN_REALTIME ]: scanIncludesInfo,
+		[ PRODUCT_JETPACK_SCAN_REALTIME_MONTHLY ]: scanIncludesInfo,
+		[ PRODUCT_JETPACK_SEARCH ]: searchIncludesInfo,
+		[ PRODUCT_JETPACK_SEARCH_MONTHLY ]: searchIncludesInfo,
+		[ PRODUCT_JETPACK_BOOST ]: boostIncludesInfo,
+		[ PRODUCT_JETPACK_BOOST_MONTHLY ]: boostIncludesInfo,
+		[ PRODUCT_JETPACK_SOCIAL_BASIC ]: socialIncludesInfo,
+		[ PRODUCT_JETPACK_SOCIAL_BASIC_MONTHLY ]: socialIncludesInfo,
+	};
+};
+
+export const getJetpackProductsBenefits = (): Record< string, Array< TranslateResult > > => {
+	const backupBenefits = [
+		translate( 'Protect your revenue stream and content' ),
+		translate( 'Restore your site in one click from desktop or mobile' ),
+		translate( 'Restore or clone offline sites' ),
+		translate( 'Fix your site without a developer' ),
+		translate( 'Protect Woo order and customer data' ),
+		translate( 'Best-in-class support from WordPress experts' ),
+	];
+
+	const scanBenefits = [
+		translate( 'Protect your revenue stream and content' ),
+		translate( 'Learn about issues before your customers are impacted' ),
+		translate( 'Fix most issues in one click from desktop or mobile' ),
+		translate( 'Best-in-class support from WordPress experts' ),
+	];
+	const antiSpamBenefits = [
+		translate( 'Set up in minutes without a developer' ),
+		translate( 'Save time manually reviewing spam' ),
+		translate( 'Increase engagement by removing CAPTCHAs' ),
+		translate( 'Best-in-class support from WordPress experts' ),
+	];
+	const videoPressBenefits = [
+		translate( 'Increase engagement and get your message across' ),
+		translate( 'Drag and drop videos through the WordPress editor' ),
+		translate( 'Manage videos in the WordPress media library' ),
+		translate( 'Remove distracting ads' ),
+		translate( 'Customizable colors to fit your brand and site' ),
+		translate( 'Best-in-class support from WordPress experts' ),
+	];
+
+	const searchBenefits = [
+		translate( "Customizable to fit your site's design" ),
+		translate( 'Increase conversion with accurate search results' ),
+		translate( 'Tiered pricing - pay for only what you need' ),
+		translate( 'No developer required' ),
+		translate( 'Best-in-class support from WordPress experts' ),
+	];
+
+	const boostBenefits = [
+		translate( 'Quickly test and improve your site speed' ),
+		translate( "Improve your site's SEO" ),
+		translate( 'Get faster FCP and LCP' ),
+		translate( 'No developer required' ),
+		translate( 'Best-in-class support from WordPress experts' ),
+	];
+
+	const socialBenefits = [
+		translate( 'Save time by sharing your posts automatically' ),
+		translate( 'Unlock your growth potential by building a following on social' ),
```

Comment:
Are we missing the word "media" at the end?

I think it should be:

"... building a following on social **_media_**"